src/lib/github/types.ts

```diff
@@ -8,7 +8,7 @@ export interface Context {
 
 export interface Issue {
   title: string;
-  body: string;
+  body: string | string[];
```

Comment:
`body` can't be either `string | string[]`  please change it back

Reply:
There is nowhere we change body to `string[]` its always `string` and I don't think it's a good idea to make it change. This is because its expected as a string by API functions that eventually use it.